### PR TITLE
#1484: Update FSPIOP API version

### PR DESCRIPTION
**Pull request by oderayi**

### CODE REVIEW

File: src/InboundServer/api.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.0
 info:
-  version: '1.0'
+  version: '1.1'
   title: Open API for FSP Interoperability (FSPIOP)
   description: >-
     Based on API Definition.docx updated on 2018-03-13 Version 1.0. Note - The
```
Comment (elnyry-sam-k) [inline src/InboundServer/api.yaml:6]:
```suggestion
    Based on API Definition.docx updated on 2020-05-19 Version 1.1. Note - The API supports a maximum size of 65536 bytes (64 Kilobytes) in the HTTP header.
```

**Comment by elnyry-sam-k** [comment]
Thanks @oderayi .. @bushjames , @msk- - any issues with upgrading to v1.1?

**Comment by bushjames** [comment]
@elnyry-sam-k @oderayi From what I see, this PR just changes the inbound swagger version to 1.1? I do not see the point as this will have no effect on the functions of the adapter at all? Incoming requests will still be validated against resource 1.0 versions. No v.1.1 functionality is supported? There are no functional changes anywhere else?

**Comment by oderayi** [comment]
Yes @bushjames. Some changes in line with the version change are coming in in a separate PR (https://github.com/mojaloop/sdk-scheme-adapter/pull/181)

**Comment by elnyry-sam-k** [comment]
Thanks @oderayi , @bushjames - this is now included in #181 that steven referenced above, maybe that makes more sense. Thanks @vgenev

**Pull request merged**